docs/core/testing/preview-changes.md

```diff
@@ -0,0 +1,29 @@
+# Preview Changes
+
+## Build
+
+### 1. Clone ionic
+
+    git@github.com:ionic-team/ionic.git
+    cd ionic
+
+### 2. Run `npm install`
+
+    cd core
+    npm install
+
+Notice that `@ionic/core` lives in `core`.
+
+### 3. Run `npm start`
+
+Make sure you are inside the `core` directory.
+
+    npm start
+
+With the `dev` command, Ionic components will be built with [Stencil](https://stenciljs.com/), changes to source files are watched, a local http server will startup, and [http://localhost:3333/](http://localhost:3333/) will open in a browser.
```

Comment:
Sorry what do you mean? They render as anchors here: https://github.com/ionic-team/ionic-framework/blob/b9634661198b036c87b2f65da516aa69d497d4b5/docs/core/testing/preview-changes.md#3-run-npm-start

Reply:
As in the text for the link should be readable words instead of a URL:
`a local http server will startup, and will open in a [browser](http://localhost:3333/).`

Reply:
I don't see any axe errors for these links here: https://brandyscarney.github.io/docs-consolidation/core/testing/preview-changes.html

I think the documentation here wanted the URL to be visible. I can remove the links if we don't want them but I think we should keep the localhost visible. This can also be updated later when we improve the documentation.

Reply:
Oh, I didn't realize that [Axe doesn't check for that](https://dequeuniversity.com/checklists/web/links). It's always been a good practice to [not use URLs](https://www.nysed.gov/webaccess/create-accessible-hypertext-links) as the text that I assumed that it was a rule.

Reply:
Yeah we can improve it more when we get these published to a site. I just want to get them in one place. 🙂